README.md

```diff
@@ -2,41 +2,40 @@
 
 **Starter project enabling you to add multi-room audio streaming via Bluetooth, Airplay or Spotify Connect to any old speakers or Hi-Fi using just a Raspberry Pi.**
 
-# Highlights
+## Highlights
 
 - **Bluetooth, Airplay, Spotify Connect and UPnP**: Stream audio from your favourite music services or directly from your smartphone/computer using bluetooth or UPnP.
 - **Multi-room synchronous playing**: Play perfectly synchronized audio on multiple devices all over your place.
 - **Extended DAC support**: Upgrade your audio quality with one of our supported DACs
 
-# Setup and configuration
+## Setup and configuration
 
 Running this project is as simple as deploying it to a balenaCloud application. You can do it in just one click by using the button below:
 
-[![](https://balena.io/deploy.png)](https://dashboard.balena-cloud.com/deploy)
+[![deploy button](https://balena.io/deploy.png)](https://dashboard.balena-cloud.com/deploy)
 
-# Documentation 
+## Documentation
 
 Head over to our [docs](https://sound.balenalabs.io/docs/) for detailed installation and usage instructions, customization options and more!
 
-# Motivation
+## Motivation
 
-![](https://raw.githubusercontent.com/balenalabs/balena-sound/master/docs/images/sound.png)
+![concept](./docs/images/sound.png)
```

Comment:
Reverted this URL because of landr too. It won't find the image with a relative path unfortunately.